docs/validators/validator-setup.md

```diff
@@ -79,7 +84,7 @@ To be in the validator set, you need to have more total voting power than the 10
 
 ### Problem #1: My validator has `voting_power: 0`
 
-Your validator has become auto-unbonded. In `gaia-6002`, we unbond validators if they do not vote on `50` of the last `100` blocks. Since blocks are proposed every ~2 seconds, a validator unresponsive for ~100 seconds will become unbonded. This usually happens when your `gaiad` process crashes.
+Your validator has become auto-unbonded. In `gaia-7001`, we unbond validators if they do not vote on `50` of the last `100` blocks. Since blocks are proposed every ~2 seconds, a validator unresponsive for ~100 seconds will become unbonded. This usually happens when your `gaiad` process crashes.
```

Comment:
`gaia-7001` has changed the `offline for 50 out of 100 blocks == slash` to a different, more lenient value. I can't remember what it was, is it offline for 2000 out of the last 4000 blocks?

Reply:
20,000 out of 40,000

Reply:
what about the following piece?

> blocks are proposed every ~2 seconds, a validator unresponsive for ~100 seconds will become unbonded